### PR TITLE
#GraphQL server

### PR DESCRIPTION
**Pull request by VovaSanduliak**
dev
GraphQL server side

* [Main GitHub Project ticket](https://github.com/EventsExpress/EventsExpress/projects/2)

## Code reviewers

@sand0
@iuriikhrystiuk

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EventsExpress.Core/EventsExpress.Core.csproj [hunk 1]
```diff
@@ -30,4 +36,11 @@
     <ProjectReference Include="..\EventsExpress.Db\EventsExpress.Db.csproj" />
   </ItemGroup>
 
+  <ItemGroup>
```
Comment (iuriikhrystiuk) [inline EventsExpress.Core/EventsExpress.Core.csproj]: Remove this section please.

File: EventsExpress.Core/EventsExpress.Core.csproj [hunk 2]
```diff
@@ -10,9 +10,14 @@
     <PackageReference Include="AutoMapper.Extensions.Microsoft.DependencyInjection" Version="8.1.1" />
     <PackageReference Include="FreeImage-dotnet-core" Version="4.3.6" />
     <PackageReference Include="GeoAPI.CoordinateSystems" Version="2.0.0-nuget005" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.10.0" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.11.0-beta.1" />
```
Comment (iuriikhrystiuk) [inline EventsExpress.Core/EventsExpress.Core.csproj]: Do not use beta packages please.

File: .gitignore [hunk 3]
```diff
@@ -280,4 +280,5 @@ __pycache__/
 /EventsExpress/Properties/PublishProfiles/*
 /EventsExpress/ClientApp/build
 /EventsExpress/.config/dotnet-tools.json 
-/.vscode/*
\ No newline at end of file
+/.vscode/*
+/EventsExpress.Test/GraphQLTests/__snapshots__/EventTests.GetEvents.snap
```
Comment (iuriikhrystiuk) [inline .gitignore]: Remove the folder and not individual files please.

File: EventsExpress.Db/EventsExpress.Db.csproj [hunk 4]
```diff
@@ -19,5 +19,11 @@
     </PackageReference>
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
   </ItemGroup>    
+  <ItemGroup>
```
Comment (iuriikhrystiuk) [inline EventsExpress.Db/EventsExpress.Db.csproj]: Undo all stylecop usage. We have directory.build.props for this purpose.

File: EventsExpress/Startup.cs [hunk 5]
```diff
@@ -1,47 +1,48 @@
...
+    using MicroElements.Swashbuckle.FluentValidation.AspNetCore;
+    using Microsoft.AspNetCore.Authentication.JwtBearer;
+    using Microsoft.AspNetCore.Authorization;
+    using Microsoft.AspNetCore.Builder;
+    using Microsoft.AspNetCore.Hosting;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Localization;
+    using Microsoft.AspNetCore.SignalR;
+    using Microsoft.AspNetCore.SpaServices.ReactDevelopmentServer;
+    using Microsoft.EntityFrameworkCore;
```
Comment (iuriikhrystiuk) [inline EventsExpress/Startup.cs]: Move usings outside of namespace.

File: EventsExpress/appsettings.json [hunk 6]
```diff
@@ -15,7 +15,8 @@
   },
   "Logging": {
     "LogLevel": {
-      "Default": "Warning"
+      "Default": "Warning",
+      "Microsoft.EntityFrameworkCore": "Debug"
```
Comment (iuriikhrystiuk) [inline EventsExpress/appsettings.json]: Move this setting to apsettings.development.json please.

**Review by iuriikhrystiuk — COMMENTED** [review]
Please undo any changes to HandlerTests files.

**Review by sand0 — DISMISSED** [review]
LGTM

File: EventsExpress.Core/EventsExpress.Core.csproj (hunk 1 above)
Comment (idonotexisttoday) [reply EventsExpress.Core/EventsExpress.Core.csproj]: Done

File: EventsExpress.Core/EventsExpress.Core.csproj (hunk 2 above)
Comment (idonotexisttoday) [reply EventsExpress.Core/EventsExpress.Core.csproj]: Done

File: .gitignore (hunk 3 above)
Comment (idonotexisttoday) [reply .gitignore]: Done

File: EventsExpress/Startup.cs (hunk 5 above)
Comment (idonotexisttoday) [reply EventsExpress/Startup.cs]: Done

File: EventsExpress/appsettings.json (hunk 6 above)
Comment (idonotexisttoday) [reply EventsExpress/appsettings.json]: Done

File: EventsExpress.Db/EventsExpress.Db.csproj (hunk 4 above)
Comment (idonotexisttoday) [reply EventsExpress.Db/EventsExpress.Db.csproj]: Done

File: EventsExpress.Core/EventsExpress.Core.csproj (hunk 1 above)
Comment (iuriikhrystiuk) [reply EventsExpress.Core/EventsExpress.Core.csproj]: But you did not remove it :)

File: EventsExpress.Core/Services/BookmarkService.cs [hunk 7]
```diff
@@ -11,38 +11,40 @@ namespace EventsExpress.Core.Services;
 
 public class BookmarkService : IBookmarkService
```
Comment (iuriikhrystiuk) [inline EventsExpress.Core/Services/BookmarkService.cs]: Undo this file please.

File: EventsExpress.Test/EventsExpress.Test.csproj [hunk 8]
```diff
@@ -15,7 +15,7 @@
     <PackageReference Include="Moq" Version="4.16.1" />
     <PackageReference Include="nunit" Version="3.13.2" />
     <PackageReference Include="NUnit3TestAdapter" Version="4.1.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0-preview-20211109-03" />
```
Comment (iuriikhrystiuk) [inline EventsExpress.Test/EventsExpress.Test.csproj]: Undo this file please.

File: EventsExpress/EventsExpress.csproj [hunk 9]
```diff
@@ -24,7 +24,7 @@
   <ItemGroup>
```
Comment (iuriikhrystiuk) [inline EventsExpress/EventsExpress.csproj]: Undo this file as well

**Review by iuriikhrystiuk — APPROVED** [review]

**Pull request merged**